### DISCUSSION TITLE
"Only you can see your full profile" and "This is your contribution graph" profile page messages

### DISCUSSION THREAD

**Discussion by dandev486** [community/community#46258] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, I would like to provide feedback on two non-dismissable messages in the profile page. In both cases, the suggestion is either to add a dismiss button or to add an expiration period, so that after the user is aware of the contents, they are no longer displayed.

- This message is displayed after making the profile private. It is informative for the first time when the feature is enabled, but there is no "dismiss button" and thus it is always visible after that.

![20230203072538](https://user-images.githubusercontent.com/29714040/216576607-c0ba280f-6007-4ebc-86ae-16274d3247d0.png)

- This message is displayed below the contribution graph. It is informative for new users, but there is no "dismiss button" and thus it is always visible.

![20230203072546](https://user-images.githubusercontent.com/29714040/216576613-49228ba0-ffef-48a6-950d-ccaf895ffff6.png)

In time, this profile is not new (> 5 years), so these messages seem not to be displayed only in new profiles, but rather on profiles that were changed to be private.

Thanks!

**Comment by jge162** [comment]
these go away once you start using your account.

**Reply by dandev486** [reply]
I think that this is not the case, since I've been using my account since 2017 and have a few private repositories and just recently activated the private profile.

**Reply by jge162** [reply]
>This message is displayed after making the profile private. 

I think it is tied to being private as you mentioned.  Nevertheless, good suggestion to be able dismiss messages.

**Reply by dandev486** [reply]
I tried creating a public repository just to check if that was the issue - even though I've already had public repositories before - and the messages are still visible, for instance:

![20230203075647](https://user-images.githubusercontent.com/29714040/216586229-603c2906-8e0e-477a-aa94-6bae809c96bd.png)

![20230203075620](https://user-images.githubusercontent.com/29714040/216586225-3c021c40-7f8c-49b5-ba71-8c9edc368ed8.png)

**Reply by jge162** [reply]
Maybe it is a bug? You can submit a [bug report](https://support.github.com/contact/bug-report)?

**Reply by dandev486** [reply]
I pondered the possibility and probably will try that later on, but first I'd like - to some extent - make sure that the behaviour isn't intentional prior to reporting it as a bug.

**Reply by jge162** [reply]
Understandable. Others might reply here too and you can get a consensus prior to submitting a bug report.

**Reply by grtcdr** [reply]
I would also like to be able to dismiss this message.

**Reply by aprofessionalusername** [reply]
Same

**Comment by dandev486** [comment]
[Similar Report](https://github.com/orgs/community/discussions/116775).
